loopchain/channel/channel_service.py

```diff
@@ -532,7 +564,7 @@ def __subscribe_to_peer_list(self):
                 loopchain_pb2.PeerRequest(
                     channel=ChannelProperty().name,
                     peer_target=ChannelProperty().peer_target, peer_type=peer_type,
-                    peer_id=ChannelProperty().peer_id, group_id=ChannelProperty().group_id,
+                    peer_id=ChannelProperty().peer_id, group_id=ChannelProperty().peer_id,
```

Comment:
This group_id should be deleted.

Reply:
`group_id` is partially removed here. Group ids related to grpc proto will be updated at the next pull request.